src/app/AppMeshSelector.vue

```diff
@@ -51,9 +50,14 @@ function changeMesh(event: Event): void {
 
   store.dispatch('updateSelectedMesh', mesh)
 
-  const name = 'mesh' in route.params ? route.name as string : 'mesh-detail-view'
-
-  router.push({ name, params: { mesh } })
+  router.push(
+    {
+      name: 'mesh-overview-view',
```

Comment:
**Question**: Why did you remove the functionality to stay on the current page if the route is a mesh route?

Reply:
Based on the meeting we had last week where John said it was more useful to land on the overview page everytime we select a Mesh (this was when I asked that we were loosing this by moving to a list mesh selection rather than a dropdown).

I believe we are removing this dropdown menu very soon anyway?

Reply:
> I believe we are removing this dropdown menu very soon anyway?

That’s true.